tools/c7n_azure/c7n_azure/query.py

```diff
@@ -48,5 +48,30 @@ def __new__(cls, name, parents, attrs):
 
 
 @six.add_metaclass(QueryMeta)
-class QueryResourceManager(object):
-    pass
+class QueryResourceManager(ResourceManager):
+
+    def __init__(self, data, options):
+        super(QueryResourceManager, self).__init__(data, options)
+        self.source = self.get_source(self.source_type)
```

Comment:
When I try to validate a policy with tag filters I get the error...
```
2018-03-26 21:58:42,286: custodian.commands:ERROR Configuration invalid: policy.yml
2018-03-26 21:58:42,286: custodian.commands:ERROR Policy: test-azure is invalid: 'NoneType' object is not callable
```

There may be some other missing configurations as well.